packages/edit-site/src/components/global-styles/use-global-styles-output.js

```diff
@@ -449,17 +449,19 @@ export const getNodesWithStyles = ( tree, blockSelectors ) => {
 	}
 
 	const pickStyleKeys = ( treeToPickFrom ) =>
-		pickBy( treeToPickFrom, ( value, key ) =>
-			[
-				'border',
-				'color',
-				'dimensions',
-				'spacing',
-				'typography',
-				'filter',
-				'outline',
-				'shadow',
-			].includes( key )
+		Object.fromEntries(
+			Object.entries( treeToPickFrom ?? {} ).filter( ( [ key ] ) =>
```

Comment:
Theoretically, `treeToPickFrom` should not be nullish, but in practice, since `getNodesWithStyles` is an externally-exposed API, it could be nullish, so I'm going on the safe side here and defaulting to an empty object just in case.